### PR TITLE
Upgrade cardano-crypto-wrapper to depend on GHC 8.6.3

### LINKED ISSUE #265: Upgrade cardano-crypto-wrapper to GHC 8.6.3
To be able to integrate with other packages that are part of the Shelley rewrite, `cardano-crypto-wrapper` has to be upgraded from GHC 8.4.4 to GHC 8.6.3.

### PR DESCRIPTION
**Pull request by mdimjasevic**
This upgrades `cardano-crypto-wrapper`'s stack build to GHC 8.6.3.

The `tasty-hedhehog` package had to be declared in a `stack.yaml` file.

Closes #265.

### CODE REVIEW

File: crypto/stack.yaml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
   - git: https://github.com/input-output-hk/cardano-prelude
-    commit: 54f2861978a02a2b04f626f47005769bf23b853b
+    commit: 3c5b1f78bb07d361ff8a1d20bf0350d0c5b115b9
     subdirs:
       - .
       - test
 
   - ../binary
   - ../binary/test
+  - pretty-show-1.8.2
```
Comment (ruhatch) [inline crypto/stack.yaml]: Shouldn't be necessary if you update the snapshot in `cardano-prelude`

**Review by ruhatch — CHANGES_REQUESTED** [review]

File: crypto/stack.yaml (hunk 1 above)
Comment (mdimjasevic) [reply crypto/stack.yaml]: @ruhatch, you're right. I removed this.

**Comment by mdimjasevic** [comment]
bors try

**Comment by mdimjasevic** [comment]
@ruhatch, can you make your review approving? Your concern was addressed and bors builds pass now.

**Review by ruhatch — APPROVED** [review]

**Pull request merged**